CONTRIBUTING.md

```diff
@@ -123,22 +123,22 @@ To run all tests for the `conjur-env` Golang module *and* for
 `lib/0001_retrieve-secrets.sh`, you can run:
```

Comment:
`lib/0001_retrieve-secrets.sh` should be `buildpack/lib/0001_retrieve-secrets.sh`?